inquestlabs.py

```diff
@@ -134,7 +134,7 @@ def __init__ (self, api_key=None, config=None, proxies=None, base_url=None, retr
             # NOTE: if we still don't have an API key that's fine! InQuest Labs will simply work with some rate limits.
 
     ####################################################################################################################
-    def __API (self, api, data=None, path=None, method="GET", raw=False):
+    def API (self, api, data=None, path=None, method="GET", raw=False):
```

Comment:
Now that we *have* to access this routine outside of the class, let's use ._API() to at least hint to folks that this is an internal routine and not to be called directly.

Reply:
Double underscore and single underscore cause issues for me when using mock.patch(). 

(This works)
    mocker.patch("inquestlabs.inquestlabs_api.API", return_value=mock_response)

(This doesn't)
     mocker.patch("inquestlabs.inquestlabs_api._API", return_value=mock_response)

(neither does this)
    mocker.patch("inquestlabs.inquestlabs_api.__API", return_value=mock_response)

Reply:
This is one solution I was looking at to keep private functions, but I have not been able to get it to work. However, this is supposedly valid:

```python
def test_mock_private(self):
        c = Car()
        with mock.patch.object(c, '_Car__private', return_value=3) as method:
            c.no_private()
            method.assert_called_once_with()
```

```python
class Car:
    def __private(self):
        return 1
    def no_private(self):
        return self.__private()
```

Reply:
More trouble than it's worth. We can go with non-private. We'll use the ALL_CAPS to differentiate API() and HASH() from the rest and we can add a small comment to the function documentation about not calling these functions directly.